pkg/kubelet/kubelet.go

```diff
@@ -179,6 +181,15 @@ const (
 	minDeadContainerInPod = 1
 )
 
+var etcHostsPath = getEtcHostsPath()
+
+func getEtcHostsPath() string {
```

Comment:
Unfortunate overlap with the same function name in kubelet_pods.go that returns the _host-side_ path.

Reply:
And after rebasing, this is now causing a build failure. I guess `getEtcHostsPath` was added to kubelet_pods.go since this PR was originally written.